core/rpc/src/Mu/Rpc/Examples.hs

```diff
@@ -9,7 +10,7 @@
 {-# language PolyKinds             #-}
 {-# language TypeFamilies          #-}
 {-# language TypeOperators         #-}
-{-# OPTIONS_GHC -fno-warn-orphans #-}
+{-# OPTIONS_GHC -fno-warn-partial-type-signatures #-}
 module Mu.Rpc.Examples where
```

Comment:
Maybe we should move this example to the `examples` folder? 🤔

Reply:
I like to have them here as a small examples people can see while browsing the API docs.